data-pipeline.yml

```diff
@@ -0,0 +1,222 @@
+version: 1
+tasks:
+  - ID: commit-retrieval
+    created: {$fromNow: ''}
+    deadline: {$fromNow: '1 hour'}
```

Comment:
This task will very likely take longer than 1 hour, I think we should be safe with 12 hours.